testsuite/tests/warnings/w53.ml

```diff
@@ -378,3 +378,9 @@ end
 
 (* Attributes in attributes shouldn't be tracked for w53 *)
 [@@@foo [@@@deprecated]]
+
+module TestNewtypeAttr = struct
+  (* Check for handling of attributes on Pexp_newtype *)
+  let f1 = fun [@inline] (type a) (x : a) -> x (* accepted *)
```

Comment:
Is it worthwhile / possible to test that the attribute actually does a thing?

Reply:
Probably possible, but I think the actual semantic behavior of attributes should probably be tested by attribute-specific tests.  This is a test for warning 53 - the mechanism for giving warnings if the compiler never looks at your attribute.  To figure out how to check that it subsequently does the right thing for each of the ~couple dozen built-in attributes in each of the positions they occur would be a lot of work.

Reply:
Fair enough.